.pre-commit-config.yaml

```diff
@@ -35,13 +39,13 @@ repos:
       entry: cmd.exe /c "scons checkPot -j 4"
       language: system
       pass_filenames: false
-      types: [python, pofile]
+      types: [python]
     - id: unitTest
       name: unit tests
       entry: ./rununittests.bat
       language: script
       pass_filenames: false
-      types: [python, c, c++, batch]
+      types_or: [python, c, c++, batch]
```

Comment:
I wonder whether it makes sense to run the tests when c/c++ files have changed. It is necessary to run scons source first when that's the case. May be we should add that? What do you think @seanbudd?

Reply:
Good question - I'm not sure. I think we need to run tests when certain dependencies change - e.g. the braille tables test when liblouis changes. This probably requires a scons source

Reply:
I think adding scons source to pre-commit is a good idea for c++ changes, to ensure NVDA builds successfully